.ci/containers/terraform-vcr-tester/run_vcr_tests.sh

```diff
@@ -27,6 +27,24 @@ else
     echo "Running tests: Go files changed"
 fi
 
+function add_comment {
+      curl -H "Authorization: token ${GITHUB_TOKEN}" \
+        -d "$(jq -r --arg comment "${1}" -n "{body: \$comment}")" \
+        "https://api.github.com/repos/GoogleCloudPlatform/magic-modules/issues/${2}/comments"
+}
+
+$BRANCHNAME=$(echo -n "/auto-pr-${pr_number}"| base64)
+curl --header "Accept: application/json" --header "Authorization: Bearer $TEAMCITY_TOKEN" https://ci-oss.hashicorp.engineering/app/rest/builds/multiple/branch:name:\(\$base64:$BRANCHNAME\),buildType:id:GoogleCloudBeta_ProviderGoogleCloudBetaMmUpstreamVcr,property:\(name:env.VCR_MODE,value:REPLAYING\),defaultFilter=false --request POST --header "Content-Type:application/xml" --data-binary @/teamcitycancelparams.xml -o result.json
+
+ERRS=$(cat result.json | jq .errorCount -r)
+if [ "$ERRS" -gt "0" ]; then
+	for row in $(cat result.json | jq -r '.operationResult[] | @base64'); do
+	    build_url=$(${row} | base64 --decode | jq -r '.related.build.webUrl')
+	done
+	add_comment "Error trying to cancel build (${build_url})" ${pr_number}
+	exit 0
```

Comment:
Why is there an exit here? It seems like this could result in a new commit not being tested - but I may be missing something?